bin/oref0-pump-loop.sh

```diff
@@ -67,8 +67,6 @@ smb_main() {
         if grep -q '"suspended": true' monitor/status.json; then
             echo -n "Pump suspended; "
             smb_verify_status
-        else
-            echo Error, retrying && maybe_mmtune
```

Comment:
Hmm, I just noticed this.  Why are you completely disabling the ability to mmtune on failures with SMB?